### PR TITLE
Develor

### PR DESCRIPTION
**Pull request by mileantkostya**
https://mileantkostya.github.io/layout_moyo-header/

https://mileantkostya.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
first check all items from the checklist and mark in the table

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,46 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#Home" class="logo">
+        <img class="logo__png" src="images/logo.png"
+        alt="logo">
```
Comment (ManoylovAC) [inline src/index.html:19]: please check recommendations for indents [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) (13 point)

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,46 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung </a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="is-active nav__link">Smartphones </a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops & computers" class="nav__link"
+            data-qa="hover">Laptops & Computers </a>
```
Comment (ManoylovAC) [inline src/index.html:34]: not good code style as well
please fix it

File: src/style.css [hunk 3]
```diff
@@ -1 +1,71 @@
-/* Styles go here */
+
+
```
Comment (ManoylovAC) [inline src/style.css:2]: no need empty lines in start of file

File: src/style.css [hunk 4]
```diff
@@ -1 +1,71 @@
...
+* {
+  box-sizing: border-box;
+}
+html {
+  font-family: Roboto,Helvetica,sans-serif
+}
+body {
```
Comment (ManoylovAC) [inline src/style.css:9]: you should separate different rules by empty space line
it will be more clear to read
```suggestion
}

body {
```

**Review by ManoylovAC — CHANGES_REQUESTED** [review]
Hey 👋 
Good start!
Let's make some improvements.
Check out [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) and make some fixes according to code style rules.
Good luck!

**Pull request closed without merging**